### PR TITLE
Drop EOL Debian 8

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: spec/acceptance/class_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe 'squid class' do
   context 'configure http_access' do
-    squid_name = fact('operatingsystem') == 'Debian' && fact('operatingsystemmajrelease') == '8' ? 'squid3' : 'squid'
+    squid_name = 'squid'
```
Comment (ekohl) [inline spec/acceptance/class_spec.rb:5]: Perhaps the whole variable should go?

Comment (bastelfreak) [reply spec/acceptance/class_spec.rb]: fixed it

**Review by igalic — APPROVED** [review]

**Comment by igalic** [comment]
are we going to fix Tests in a different PR?

File: manifests/params.pp [hunk 2]
```diff
@@ -43,22 +43,11 @@
...
-        }
-        default: {
-          $package_name          = 'squid'
-          $service_name          = 'squid'
-          $config                = '/etc/squid/squid.conf'
-          $access_log            = 'daemon:/var/log/squid/access.log squid'
-          $squid_bin_path        = '/usr/sbin/squid'
-        }
-      }
+      $package_name          = 'squid'
```
Comment (ekohl) [inline manifests/params.pp:46]: Do we want to pull out the ones that are always the same for all 3 branches of the `case`?

**Review by ekohl — APPROVED** [review]

File: manifests/params.pp [hunk 3]
```diff
@@ -40,52 +40,30 @@
...
-          $squid_bin_path        = '/usr/sbin/squid'
-        }
-      }
-
-      $config_user           = 'root'
-      $config_group          = 'root'
-      $daemon_user           = 'proxy'
-      $daemon_group          = 'proxy'
+      $config       =  '/etc/squid/squid.conf'
+      $config_group =  'root'
```
Comment (ekohl) [inline manifests/params.pp:52]:
```suggestion
      $config       = '/etc/squid/squid.conf'
      $config_group = 'root'
```

File: manifests/params.pp (hunk 2 above)
Comment (bastelfreak) [reply manifests/params.pp]: done

**Review by ekohl — APPROVED** [review]

**Pull request merged**